packages/outputarea/src/widget.ts

```diff
@@ -250,6 +251,19 @@ class OutputArea extends Widget {
     }
   }
 
+  /**
+   * Follow changes on the model state.
+   */
+  protected onModelStateChanged(sender: IOutputAreaModel, model: IOutputModel): void {
+    let layout = this.layout as PanelLayout;
+    layout.widgets.forEach((panel: Panel) => {
+      let renderer = (panel.widgets ? panel.widgets[1] : panel) as IRenderMime.IRenderer;
```

Comment:
What is the purpose of this check, here and below?

Reply:
If you create a new view for an output and then call an `IPython.display.update_display` on that output, it will throw an error because for some reason, that _output panel_ doesn't have any `widgets` property.